js/scripts.js

```diff
@@ -89,7 +89,7 @@ const calculateOutput = function (data) {
     var out_line = "<tr><td>" + poss.pattern_description + "</td>"
```

Comment:
```suggestion
    var out_line = "<tr><td class='table-pattern'>" + poss.pattern_description + "</td>"
```

Allows us to style this element specifically